app/views/layouts/_notifications_drawer.html.haml

```diff
@@ -0,0 +1,17 @@
+%div{"id" => "miq-notifications-drawer-container", 'ng-controller' => "notificationsDrawerController as vm"}
+  %div{"pf-notification-drawer" => "",
+       "id" => "miq-notifications-drawer",
+       "drawer-hidden" => "!vm.notificationsDrawerShown",
+       "drawer-title" => "Notifications",
+       "allow-expand" => "true",
+       "action-button-title" => "Mark All Read",
+       "action-button-callback" => "vm.markAllRead",
+       "notification-groups" => "vm.notificationGroups",
+       "heading-include" => "/static/notification_drawer/notification-heading.html",
+       "subheading-include" => "/static/notification_drawer/notification-subheading.html",
+       "notification-body-include" => "/static/notification_drawer/notification-body.html",
+       "notification-footer-include" => "/static/notification_drawer/notification-footer.html",
+       "custom-scope" => "vm.customScope"}
+
+:javascript
+  miq_bootstrap('#miq-notifications-drawer-container');
```

Comment:
This looks fishy.. are the notification drawer, notification header, and toast list **supposed** to be 3 different angular *applications*?